### DISCUSSION TITLE
Connection failure

### DISCUSSION THREAD

**Discussion by whairdo** [Shabinder/SpotiFlyer#141] [Q&A]
I first downloaded the Windows desktop version but failed to search anything with the message "check network connection please", then I tried the app for Android and got a "No Internet connection!" btw I kept my VPN connection on the whole time since Spotify is blocked in my country. I'm wondering if it had anything to do with the connection failure.

**Comment by Shabinder** [comment]
Which android version you tried and which Vpn , so I maybe could reproduce the issue

**Comment by liluzibird** [comment]
I'm having the same issues. You guys find a solution yet?

**Comment by Rajbir98** [comment]
bro i got no vpn but still check network connection is popping up

**Reply by down-cell** [reply]
I met Same question.have u solve it
